.babelrc

```diff
@@ -7,7 +7,8 @@
 		} ]
 	],
 	"plugins": [
-		"transform-runtime"
+		"transform-runtime",
+		"transform-object-rest-spread"
```

Comment:
I resent `class` so I'm inclined to let it be annoying to use 😆 

On a more serious note, that proposal is a little less stable (stage 2). On the flip side, I understand even just opting in to this one object feature has now opened the floodgates on what are and aren't acceptable non-spec features to include.